experimental/components/MsGraph/Actions/GetContacts.cs

```diff
@@ -90,7 +90,10 @@ protected override async Task<List<CalendarSkillContactModel>> CallGraphServiceW
 
             if (people?.Count > 0)
             {
-                var existingResult = contactsResult.SelectMany(c => c.EmailAddresses).ToHashSet();
+                var result = contactsResult.SelectMany(c => c.EmailAddresses);
+
+                var existingResult = new HashSet<string>(result);
```

Comment:
Presumably this should use StringComparer.OrdinalIgnoreCase?

Reply:
The default behavior from before does not use ordinal ignore case. We might as well fix it.